test/integration/fixtures/current-test-title.fixture.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+function getTitle() {
+  return this.currentTest ? this.currentTest.title : undefined;
```

Comment:
maybe more straightforward to skip `Function.prototype.call` altogether:

```js
function getTitle(hook) {
  return hook.currentTest && hook.currentTest.title;
}

before(function() {
  getTitle(this);
});
```

when coerced to a string you're looking at `'undefined'` in your implementation or `false` in mine; either way it should cause a test failure. you'll need to update your test using this of course